themes/ocw-prototype/LICENSE.md

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
Where did this license come from?

Licensing is a sensitive issue for course content. If the license is meant to apply to the _content_ then it should a) be a Creatitive Commons CC-BY-NC-SA by default and b) we'd like to let course authors determine the license -- possibly at a per resource level.

Reply:
You added this to the project with the initial commit.

Reply:
Sorry, looking at this again you clearly did not add it.  I thought this was the license under the root of the project, but looking at it again it came from a boilerplate theme that I copied into this branch and modified.